hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/FederationApplicationHomeSubClusterStore.java

```diff
@@ -117,5 +117,4 @@ GetApplicationsHomeSubClusterResponse getApplicationsHomeSubCluster(
    */
   DeleteApplicationHomeSubClusterResponse deleteApplicationHomeSubCluster(
       DeleteApplicationHomeSubClusterRequest request) throws YarnException;
-
```

Comment:
Let's avoid this change.

Reply:
Thank you very much for your help in reviewing the code! I will modify the code.